tests/unit/drivers/complex_vlan.py

```diff
@@ -43,6 +43,9 @@ def config_initialize():
                         '{"name":"1", "switch":"test"}, ' \
                         '{"name":"2", "switch":"test"}]')
             cfg.set('driver complex_vlan', 'trunk_ports', '[]')
+            cfg.add_section('database')
+            cfg.set('database', 'sqlite', 'sqlite:///:memory:')
+            cfg.set('database', 'postgres', 'postgresql://postgres:postgres@localhost/haas')
```

Comment:
"If somebody wants to test HaaS code with postgresql, they should edit this information everywhere." - Should I add this information in some doc file?